README.md

```diff
@@ -42,100 +41,83 @@ You will then need to add a migration:
           add :claims, :map
           timestamps()
         end
-
       end
-
     end
 ```
 
-**Do not run the migration yet.** You also need to add this to your configuration:
+**Do not run the migration yet.** You also need to finish your setup first.
```

Comment:
This doesn't read quite right, what about something like:

**Do not run the migration yet**, we need to complete our setup first: